ntc_templates/templates/cisco_nxos_show_ip_bgp.textfsm

```diff
@@ -1,5 +1,5 @@
 # Status: s-suppressed, x-deleted, S-stale, d-dampened, h-history, *-valid, >-best
-Value STATUS ([sxSdh*])
+Value STATUS ([sxSdh*\s])
```

Comment:
Instead of this, add this before ` ^\S+${PATH_SELECTION}${ROUTE_SOURCE}\s\s\s{16}(\s{0,12}${NEXT_HOP}\s{4}|\s{20})(\s{2,14}${METRIC}|\s{8})(\s{1,18}${LOCAL_PREF}|\s{8})(\s{1,22}${WEIGHT})(\s${AS_PATH})?\s${ORIGIN}$$ -> Record
`

You may need to do this for both formats `${NETWORK}` and `\s\s\s{16}`

Reply:
Updated, please take a look if you don't mind @itdependsnetworks